src/commands/push-to-docs-gcs.yml

```diff
@@ -0,0 +1,52 @@
+description: |
+  Uses rsync to push a selected folder to Entur Docs - Google Cloud Storage.
+
+parameters:
+  gcp-service-key:
+    description: |
+      The credentials for the Service Account to be used when communicating to Google Cloud.
+      If not provided, the command will look for the environment variable $DOCS_PASSWORD
+
+      To obtain the key for this command,
+      use either the Google Cloud Platform Console or gcloud iam service-accounts keys create.
+      The key can be .json (preferred) or .p12 (legacy) format.
+    type: string
+    default: ""
+  project-name:
+    description: |
+      Specify the project name (e.g. mobility, customers). This is the same as the bucket name in GCS.
+    type: string
+  docs-path:
+    description: |
+      Specify the folder which contains .mdx and other required files used by the developer portal, without trailing slashes (e.g. docs, ./docs, .)
+      Default is 'docs'
+    type: string
+    default: "docs"
+
+steps:
+  - authenticate-gcp:
+      gcp-service-key: << parameters.gcp-service-key >>
+  - run:
+      name: Upload docs to Entur Docs GCS
+      command: |
+        PROJECT_NAME="<< parameters.project-name >>"
+        DOCS_FOLDER="<< parameters.docs-path >>"
+
+        if [ -z "${PROJECT_NAME}" ]; then
+          echo "Project name must be specified."
+          exit 1
+        fi
+
+        if ! which rsync > /dev/null; then
+          echo "rsync is not installed. Please run this command with a executor that supports rsync."
+          exit 1
+        fi
+
+        echo "Preparing working directory..."
+        rsync -R ${DOCS_FOLDER}/** temp-docs -v
```

Comment:
Du burde nok slenge på `.mdx` så ikkje alle mulige andre filer blir sendt med.
Slik:
```
rsync -R ${DOCS_FOLDER}/**.mdx temp-docs -v
```

Reply:
Hei! Ja, jeg tenkte på det, men vi har en `.png` fil vi gjerne vil ha med i dokumentasjonen. Jeg ser for meg at andre også vil ønske å laste opp flere ting etterhvert.. (?)

Reply:
Ah, godt poeng. Ja, då er eg einig at det er best å ha ei eiga mappe for dokumentasjon fram til det eventuelt blir behov for noko anna.